src/pages/[entity]/index.tsx

```diff
@@ -1,24 +1,21 @@
-import React, { useState } from "react";
-
+import { Badge } from "@material-ui/core";
```

Comment:
Let's avoid this! Always get the component from `/Component` in material UI

```typescript
import Badge from "@material-ui/core/Badge";
```

Reply:
Same comment from previous: organize the imports

Reply:
by doing the import without  `{}`:
```ts
import Badge from "@material-ui/core/Badge";
```

gave error when using the element:
```
import Badge
JSX element type 'Badge' does not have any construct or call signatures.ts(2604)
```